.github/workflows/generate-migration-sql.yaml

```
@@ -0,0 +1,98 @@
+# Generate migration as SQL if necessary
+# https://github.com/actions/github-script#comment-on-an-issue
+
+name: "Migration as SQL"
+
+on:
+  workflow_dispatch:
+  pull_request:
+    paths:
+      - alembic/versions/**
+
+jobs:
+  comment:
+    runs-on: ubuntu-latest
```

`suggestion(blocking)`:
```suggestion
    runs-on: ubuntu-22.04
```